lib/texture1.c

```diff
@@ -1323,11 +1323,19 @@ struct ktxTexture_vtbl ktxTexture1_vtbl = {
     (PFNKTEXITERATELOADLEVELFACES)ktxTexture1_IterateLoadLevelFaces,
     (PFNKTEXNEEDSTRANSCODING)ktxTexture1_NeedsTranscoding,
     (PFNKTEXLOADIMAGEDATA)ktxTexture1_LoadImageData,
+#if KTX_FEATURE_WRITE
     (PFNKTEXSETIMAGEFROMMEMORY)ktxTexture1_SetImageFromMemory,
     (PFNKTEXSETIMAGEFROMSTDIOSTREAM)ktxTexture1_SetImageFromStdioStream,
     (PFNKTEXWRITETOSTDIOSTREAM)ktxTexture1_WriteToStdioStream,
     (PFNKTEXWRITETONAMEDFILE)ktxTexture1_WriteToNamedFile,
     (PFNKTEXWRITETOMEMORY)ktxTexture1_WriteToMemory,
+#else
+    (PFNKTEXSETIMAGEFROMMEMORY)NULL,
+    (PFNKTEXSETIMAGEFROMSTDIOSTREAM)NULL,
+    (PFNKTEXWRITETOSTDIOSTREAM)NULL,
+    (PFNKTEXWRITETONAMEDFILE)NULL,
+    (PFNKTEXWRITETOMEMORY)NULL,
```

Comment:
Instead of NULL and therefore a crash, these should point at a stub function that returns KTX_INVALID_OPERATION.